packages/venia-ui/lib/components/ProductSort/productSort.js

```diff
@@ -15,6 +15,32 @@ const ProductSort = props => {
     const { availableSortMethods, sortProps } = props;
     const [currentSort, setSort] = sortProps;
     const { elementRef, expanded, setExpanded } = useDropdown();
+    const { locale } = useIntl();
+
+    const orderSortingList = useCallback(
+        list => {
+            return list.sort((a, b) =>
```

Comment:
Could we update the QA Steps to make sure we test this out in different languages?
What if we give it a different translation in SortItem?

Reply:
Ok, no problem. I believe `localeCompare` will order here by following language rules so it's safe.

Reply:
I had to change this to not rely on language rules, since the sort options from graphQL won't get translated.